core/domain/skill_services.py

```diff
@@ -727,6 +727,38 @@ def apply_change_list(skill_id, change_list, committer_id):
         raise e
 
 
+def populate_skill_model_with_skill(skill_model, skill):
+    """Populate skill model with the data from skill object.
+
+    Args:
+        skill_model: SkillModel. The model to populate.
+        skill: Skill. The object from which is the model populated.
```

Comment:
The skill domain object which should be used to populate the model.

(Ditto elsewhere.)

Reply:
Done.